docs/ingestion/native-batch.md

```diff
@@ -89,6 +89,8 @@ You may want to consider the below things:
   data in segments where it actively adds data: if there are segments in your `granularitySpec`'s intervals that have
   no data written by this task, they will be left alone. If any existing segments partially overlap with the
   `granularitySpec`'s intervals, the portion of those segments outside the new segments' intervals will still be visible.
+  You can set `dropExisting` flag in the `ioConfig` to true if you want the ingestion task to drop all existing data 
```

Comment:
Nit: You mentioned that is only occurs if a few other configs are set as well, appendToExisting = false, and granularitySpec set, I believe. Should we mention that here?

Reply:
Done